draginolht65n/sensor.go

```diff
@@ -83,16 +84,15 @@ func newLHT65N(
 	conf resource.Config,
 	logger logging.Logger,
 ) (sensor.Sensor, error) {
-	decoderFilePath, err := node.WriteDecoderFile(decoderFilename, decoderFile)
+	_, err := node.WriteDecoderFile(decoderFilename, decoderFile)
```

Comment:
we should still use `decoderFilePath`, in case we have to change the how the path is written in `WriteDecoderFile`.

it makes `getNodeConfig` need the input still, but I think its fine if the dragino has slightly different functions for now.